src/seedu/addressbook/AddressBook.java

```diff
@@ -427,7 +463,7 @@ private static String executeAddPerson(String commandArgs) {
         // add the person as specified
         final String[] personToAdd = decodeResult.get();
         addPersonToAddressBook(personToAdd);
-        return getMessageForSuccessfulAddPerson(personToAdd);
+        return showSuccessfullyAddedMessage(personToAdd);
```

Comment:
You have changed a couple of "getMessageXXXX" to "showXXXXMessage". However, the former is actually more accurate as the method **returns** a string, instead of **printing to output**.  :-)